finetrainers/args.py

```diff
@@ -206,7 +206,9 @@ def validate_args(args: Args):
 
 
 def _add_model_arguments(parser: argparse.ArgumentParser) -> None:
-    parser.add_argument("--model_name", type=str, required=True, choices=["ltx_video"], help="Name of model to train.")
+    parser.add_argument(
+        "--model_name", type=str, required=True, choices=["hunyuan_video", "ltx_video"], help="Name of model to train."
```

Comment:
We could determine the `choices` automatically from the config map we have right now. TODO